models/business-unit/src/company/types.ts

```diff
@@ -0,0 +1,34 @@
+import type {
+  BusinessUnitKeyReference,
+  KeyReference,
+  Company,
+  CompanyDraft,
+} from '@commercetools/platform-sdk';
+import type { TBuilder } from '@commercetools-test-data/core';
+
+//BusinessUnitDraft
+export type TCompanyDraft = CompanyDraft;
+
+//BusinessUnitDraftGraphql is only scaffolding at this time
+export type TCompanyDraftGraphql = TCompanyDraft & {
+  __typename: 'BusinessUnitDraft';
+};
```

Comment:
GraphQL drafts no longer use `__typename`

Reply:
Oops - snuck in after the rebase. Fixed with [0c72cd8](https://github.com/commercetools/test-data/pull/481/commits/0c72cd8436aafea8d4e2897fc78208e5bf5ccf05)